docs/products/postgresql/howto/identify-and-fix-pg-slow-queries.rst

```diff
@@ -0,0 +1,123 @@
+
+Under the hood, the Query Statistics tab uses the `pg_stat_statements extension <https://www.postgresql.org/docs/current/pgstatstatements.html>`_, a module that provides a means for tracking the planning and execution statistics of all SQL statements executed by your PostgreSQL® server, to provide you with the basic information that would be useful for identifying slow queries.
+
+Query Statistics
+'''''''''
+
+These are the entries provided by the Query Statistics which are deduced via the pg_stat_statements:
+
+==================      =======================================================================
+Column Type                Description
+==================      =======================================================================
+``Query``               Text of a representative statement
+``Rows``                Total number of rows retrieved or affected by the statement
+``Calls``               Number of times the statement was executed
+``Min (ms)``            Minimum time spent executing the statement, in milliseconds
+``Max (ms)``            Maximum time spent executing the statement, in milliseconds
+``Mean (ms)``           Mean time spent executing the statement, in milliseconds
+``Stddev (ms)``         Population standard deviation of time spent executing the statement, in milliseconds
+``Total (ms)``          Total time spent executing the statement, in milliseconds
+==================      =======================================================================
+
+However, creating custom queries using the ``pg_stat_statements`` yourself and leveraging all of `pg_stat_statements' view column types <https://www.postgresql.org/docs/current/pgstatstatements.html>`_ it offers might be something that helps you with your investigation use case.
+
+Pre-requisites
+'''''''''
+
+To use ``pg_stat_statements``, you'll need the pg_stat_statements extension already created which is within the :doc:`../reference/list-of-extensions` approved list, which can be done via the following ``CREATE EXTENSION`` command::
+
+  CREATE EXTENSION pg_stat_statements;
+
+
+Discovering Slow Queries
+'''''''''
+You can run the following command to display the ``pg_stat_statements`` view::
+
+   \d pg_stat_statements;
+   
+.. image:: /images/products/postgresql/pg-stat-statements-view-display.png
+   :alt: Screenshot of the pg_stat_statements view display
+.. Tip::
+
+    The following output is from PostgreSQL® ``V13``, on older versions you might find different column names such as previously ``max_time`` which is now ``max_exec_time``, always refer to the PostgreSQL® official documentation with the version you are using for accurate column matching.
+    
+You can come up with your own query to help you analyze relevant information about your recently run queries.
+
+Here is an example query that use the view to display relevant information about some previously run queries that some of them are relatively slow.
```

Comment:
I would add a title like "Example ``pg_stat_statements`` query" and reshape the phrase as following
```suggestion
The following SQL uses the ``pg_stat_statements`` view to display relevant information about the previously run queries that might be slow.
```